requirements.txt

```diff
@@ -1,5 +1,6 @@
 gevent>=1.0
-pysnmp==4.2.5
+pysnmp==4.3.5
+pysmi
```

Comment:
2nd that. We don't want pysmi and pysnmp to diverge to a point where incompatibilities might occur. Pinpointing pysmi to the corresponding pysnmp is the way to go for now :)